### PR TITLE
Add detailed info on audit log contents

### PR DESCRIPTION
**Pull request by pianohacker**
**What this PR does / why we need it**:

None of the listed audited actions had information on how to tell apart the resulting JSON log entries.

**Special notes for your reviewer**:

Not in love with the format of the example at the top, very open to ideas there.

### CODE REVIEW

**Review by xlson — APPROVED** [review]
I think this is a great improvement over what we have right now. When it comes to examples I don't really have a better idea right not but I agree, it's not super easy to follow. Might we worth adding in some complete examples instead.

File: docs/sources/enterprise/auditing.md [hunk 1]
```diff
@@ -62,62 +62,154 @@ The `additionalData` field can contain the following information:
...
+| Create alert notification | `{"action": "create", "resources": [{"type": "alert-notification"}]}` |
+| Update alert notification | `{"action": "update", "resources": [{"type": "alert-notification"}]}` |
+| Delete alert notification | `{"action": "delete", "resources": [{"type": "alert-notification"}]}` |
+
+#### Reporting
+
+| Action                    | Distinguishing fields                                     |
+| ------------------------- | --------------------------------------------------------- |
+| Create report             | `{"action": "create", "resources": [{"type": "report"}]}` |
+| Update report             | `{"action": "update", "resources": [{"type": "report"}]}` |
```
Comment (AgnesToulet) [inline docs/sources/enterprise/auditing.md:178]:
```suggestion
| Create report             | `{"action": "create", "resources": [{"type": "report"}, {"type": "dashboard"}]}` |
| Update report             | `{"action": "update", "resources": [{"type": "report"}, {"type": "dashboard"}]}` |
```

**Review by AgnesToulet — APPROVED** [review]
Looks good and great improvement to the doc!

I completed some endpoints with resources. I think you didn't add them because they are not distinguishing but I think it's useful information and for other endpoints, you put all available resources so it's more consistent that way.

**Comment by pianohacker** [comment]
Makes sense to me, thanks.

**Comment by xlson** [comment]
@pianohacker looks like this is stuck on frontend linting for some reason.

**Pull request merged**